docs/tutorial/5-relationships-and-hyperlinked-apis.md

```diff
@@ -106,6 +106,8 @@ If we're going to have a hyperlinked API, we need to make sure we name our URL p
 
 After adding all those names into our URLconf, our final `snippets/urls.py` file should look something like this:
 
+    from django.conf.urls import url, include
```

Comment:
I think here we should be adding all imports since we are saying "this is how it should be looking"